.drone.yml

```diff
@@ -7,7 +7,7 @@ build:
     - EMACS=/opt/emacs-24.5/bin/emacs
     - AKKA_TEST_TIMEFACTOR=10
   commands:
-    - if grep -rlE 'TODO|FIXME' . | grep -Ev 'target|.git|.drone.yml' > /dev/null; then
+    - if git grep -E 'TODO|FIXME' . | grep -v '.drone.yml' > /dev/null; then
```

Comment:
no need for the `.` in `git grep`. It knows where to search. Also, there is a more standard way to ignore files, see http://stackoverflow.com/a/14226610/1041691